tensorflow/contrib/layers/python/layers/layers_test.py

```diff
@@ -3451,6 +3451,30 @@ def testSpatialSoftmax(self):
       keypoints = sess.run(spatial_softmax, feed_dict)
       self.assertAllClose(keypoints, np_keypoints)
 
+  def testSpatialSoftmaxSigmoid(self):
```

Comment:
Add a test for other activation functions to prove they work too.

Reply:
What other activation functions do you suggest? 
I've really only seen sigmoid and softmax used in the literature I've read.

Reply:
I forgot to mention, softmax is already tested by the existing code below.

Reply:
Could you make that explicit in the docs?

Reply:
@sguada docstring should specify the activations now